frontend/ui/data/strings.lua

```diff
@@ -14,14 +14,17 @@ S.B_PAGE_MARGIN = _("Bottom Margin")
 S.SYNC_T_B_PAGE_MARGINS = _("Sync T/B Margins")
 S.LINE_SPACING = _("Line Spacing")
 S.COLUMNS = _("Columns")
-S.TEXT_ALIGN = _("Text Align")
+-- @translators Text alignment. Options given as icons: left, right, center, justify.
+S.TEXT_ALIGN = _("Alignment")
 S.FONTSIZE_FINE_TUNING = _("Fine Tuning")
 S.CONTRAST = _("Contrast")
+-- @translators Reflow text.
 S.REFLOW = _("Reflow")
 S.DEWATERMARK = _("Dewatermark")
 S.DOC_LANG = _("Document Language")
 S.VERTICAL_TEXT = _("Vertical Text")
 S.WORD_GAP = _("Word Gap")
+-- @translators The maximum size of a speckle to be ignored instead of being considered a character.
```

Comment:
This could probably be a little clearer.

k2pdfopt for reference:
```
-de <size>        Defect size in points.  For scanned documents, marks
                  or defects smaller than this size are ignored when bounding
                  rectangular regions.  The period at the end of a sentence is
                  typically over 1 point in size.  The default is 1.0.
```

Reply:
I kind of like it? Maybe I'd just qualify "speckle" (dust/ink?)